src/styles/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $card-border-color;
+  background: $card-background-color;
+  box-sizing: border-box;
+  padding-inline: 16px;
+
+  &__img {
+    margin-top: 33px;
+    margin-inline: 3px;
+    width: $card-img-width;
+    height: $card-img-height;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    margin-top: 40px;
+    color: $card-title-color;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    line-height: 14px;
+    color: $card-subtitle-color;
+    margin-top: 4px;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $card-title-color;
+    margin: auto 0;
+  }
+
+  &__price-text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $card-subtitle-color;
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    text-align: right;
+    color: $card-title-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__link {
+    text-transform: uppercase;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    text-decoration: none;
+    color: $card-link-color;
+    width: $card-button-width;
```

Comment:
You don`t need it here as your button always should be 100% of width, obviously given block paddings